packages/breadboard/package.json

```diff
@@ -109,6 +117,12 @@
       "output": [
         "docs/api/"
       ]
+    },
+    "test:node": {
+      "dependencies": [
+        "build:tsc"
+      ],
+      "command": "node --test dist/**/*.node.test.js"
```

Comment:
Add `"files":[], "output": []` so that we can get cache hits

Reply:
```suggestion
      "command": "node --test dist/**/*.node.test.js",
      "files":[],
      "output": []
```